cli/src/pazuzu_cli/core.clj

```diff
@@ -1,10 +1,43 @@
 (ns pazuzu-cli.core
   (:gen-class)
-  (:require [clojure.data.json :as json]))
+  (:require [pazuzu-cli.console :as console]))
 
 
+(defn create-dockerfile
+  [& [features]]
+  (println "Dockerfile created @ "
+           (->> features console/fetch-and-compile-features console/save-dockerfile)))
+
+
+; TODO: STUB!! Rewrite this function once we have function that will actually build a dockerfile.
+(def build-docker-image create-dockerfile)
```

Comment:
You can use this code: https://github.com/dryewo/docker-maker/blob/master/src/docker_maker/docker.clj